public/javascripts/templates/templates.js

```diff
@@ -0,0 +1,77 @@
+(function() {
+  var template = Handlebars.template, templates = Handlebars.templates = Handlebars.templates || {};
+templates['header'] = template({"1":function(depth0,helpers,partials,data) {
+    var stack1;
+
+  return ((stack1 = this.invokePartial(partials.logout_form,depth0,{"name":"logout_form","data":data,"indent":"\t\t","helpers":helpers,"partials":partials})) != null ? stack1 : "");
+},"3":function(depth0,helpers,partials,data) {
+    var stack1;
+
+  return ((stack1 = this.invokePartial(partials.login_form,depth0,{"name":"login_form","data":data,"indent":"\t\t","helpers":helpers,"partials":partials})) != null ? stack1 : "");
+},"compiler":[6,">= 2.0.0-beta.1"],"main":function(depth0,helpers,partials,data) {
+    var stack1;
+
+  return "<div class='navbar navbar-default'>\n	<div class='navbar-header'>\n		<img class='navbar-left' src='/images/owl.png' height=128 width=128></img>\n"
```

Comment:
Is this the only way? Keep your templates as strings all in one file? Seems messy/hard to read.